module/platform/NUSense/HardwareIO/src/HardwareIO.cpp

```diff
@@ -119,6 +119,9 @@ namespace module::platform::NUSense {
             // Battery data
             sensors->battery = 0;  // not yet implemented
 
+            // Log the latest dispatch from NUSense; only very rare exceptions should be dispatched.
+            log<NUClear::WARN>("Dispatch from NUSense: " + data.dispatch);
```

Comment:
Does this log every time we trigger on NUSense?

Reply:
Fixed so that it only logs when it has at least three characters.